.circleci/config.yml

```diff
@@ -58,20 +76,3 @@ jobs:
             else
               echo "Skipping deployment"
             fi
-      - run:
-          name: Test Nginx configuration
-          command: |
-            docker-compose exec nginx nginx -t
-            composer install -d .docker/images/nginx/tests
-            .docker/images/nginx/tests/vendor/bin/phpunit -c .docker/images/nginx/tests/phpunit.xml
-
-workflows:
```

Comment:
removed workflows as we are now using a single job that is picked up by CircleCI by default.